pkgs/development/python-modules/pandoc-latex-environment/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  poetry-dynamic-versioning,
+  panflute,
+  nemosis,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc-latex-environment";
+  version = "1.1.6.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "chdemko";
+    repo = "pandoc-latex-environment";
+    rev = "refs/tags/${version}";
+    hash = "sha256-DLuLFCKP5ip3duHlGAZnFip7GCDzWK91bka3DrknQSA=";
+  };
+
+  build-system = [ poetry-core poetry-dynamic-versioning ];
+  dependencies = [ panflute nemosis ];
```

```suggestion
  dependencies = [ panflute ];
```
unused
https://github.com/chdemko/pandoc-latex-environment/blob/a119d4c630460de5685bc17913ea5b07c6de0e11/pyproject.toml#L41-L43

---

mistake, moved to #326686